packages/office-ui-fabric-react/src/components/List/examples/List.Ghosting.Example.tsx

```diff
@@ -2,25 +2,70 @@ import * as React from 'react';
 import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
 import { List } from 'office-ui-fabric-react/lib/List';
 import { Image, ImageFit } from 'office-ui-fabric-react/lib/Image';
-import './List.Ghosting.Example.scss';
+import { mergeStyleSets, getTheme, DefaultFontStyles, FontSizes, getFocusStyle } from 'office-ui-fabric-react/lib/Styling';
 
 export type IExampleItem = { name: string; thumbnail: string };
 
 export interface IListGhostingExampleProps {
   items: IExampleItem[];
 }
 
-export class ListGhostingExample extends React.Component<IListGhostingExampleProps> {
-  constructor(props: IListGhostingExampleProps) {
-    super(props);
+const theme = getTheme();
+const classNames = mergeStyleSets({
+  container: {
+    overflow: 'auto',
+    maxHeight: 500
+  },
+  itemCell: [
+    getFocusStyle(theme, -1),
```

Comment:
[getFocusStyle](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/styles/getFocusStyle.ts) does almost exactly the same thing as the `focus-border` mixin, though with slightly different parameter defaults.